paper.tex

```diff
@@ -498,7 +498,7 @@ \subsection{List}
 
 The attribute \deff{slice} is a part of the array.
 
-The attribute \deff{sorted} is a new array with elements sorted using given comparison abstract object with two free attributes.
+The attribute \deff{sorted} is a new array with elements sorted using the .lt method of the elements passed.
```

Comment:
@EugeneDar it's better to do say:

```
using \ff{lt} attribute of the elements passed
```

it's more in line with our terminology

Reply:
@yegor256 I'll fix it.